ui/app/src/utils/token.ts

```diff
@@ -0,0 +1,7 @@
+import cookies from "js-cookie";
+import { setGlobalHeaders } from "./ajax";
+
+export default function updateToken() {
```

Comment:
- `updateToken` => `setRequestForgeryToken`
- Move into `utils/auth`. Notice there's a getRequestForgeryToken function in utils/storage already. We should rename utils/storage to utils/auth and move this function in.